tools/github-team-user-store/GitHubTeamUserStore/GitHubTeamUserStore/TeamUserGenerator.cs

```diff
@@ -0,0 +1,133 @@
+using Octokit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using GitHubTeamUserStore.Constants;
+
+namespace GitHubTeamUserStore
+{
+    public class TeamUserGenerator
+    {
+        // Team/User dictionary where the team name is the key and the list users is the value. It's worth noting
+        // that this is the list of Logins (Octokit.User.Login) which are what's used in CODEOWNERS, @mentions, etc.
+        private static Dictionary<string, List<string>> _teamUserDict = new Dictionary<string, List<string>>();
+
+        /// <summary>
+        /// Generate the team/user lists for each and every team under azure-sdk-write. Every team and user in a CODEOWNERS
+        /// file must have azure-sdk-write permissions in order to be in there which means every team/user will be under
+        /// of azure-sdk-write. This is done to limit the number of calls made because if team/user data for every team
+        /// under Azure was pulled there would be roughly 1933 teams. Getting the team/user data for that many teams would
+        /// end up taking 2500-3000 GitHub API calls whereas getting the team/user data for azure-sdk-write and its child
+        /// teams is less than 1/10th of that. The team/user data is serialized into json and stored in azure blob storage.
+        /// </summary>
+        /// <param name="gitHubEventClient">Authenticated GitHubEventClient</param>
+        /// <returns></returns>
+        public static async Task GenerateTeamUserList(GitHubEventClient gitHubEventClient)
```

Comment:
Based on this method name I would expect for it to return e.g. list of user names, but instead it returns void and has the side-effect of uploading to blob storage.

A name like `UploadTeamUserListToBlobStorage` would be better IMO.